pages/terra/policy.mdx

```diff
@@ -18,7 +18,7 @@ description: "One of Terra's main focuses is package quality. Users should be ab
 - You must have the rights or permission to redistribute the packaged software. Besides that, there is no prohibition on proprietary licenses.
 - Submissions must be maintained. We will reject submissions of unmaintained software.
 - Submissions must not be malicious.
-- Submissions should not conflict with Fedora's own packages.
+- Submissions that conflict with Fedora's own packages must be included in "Terra Extra" instead via the `extra = 1` label in `anda.hcl`.
```

Comment:
To clarify, we allow packages that conflict with Fedora (-nightly), but if the depsolver is able to resolve that package without explicit consent, that's when extra is used